src/SparkTestBase.sol

```diff
@@ -161,16 +171,40 @@ abstract contract SpellRunner is Test {
             OptimismBridgeTesting.relayMessagesToDestination(bridge, false);
         } else if (bridgeType == BridgeType.CCTP) {
             CCTPBridgeTesting.relayMessagesToDestination(bridge, false);
+            CCTPBridgeTesting.relayMessagesToSource(bridge, false);
         } else if (bridgeType == BridgeType.GNOSIS) {
             AMBBridgeTesting.relayMessagesToDestination(bridge, false);
         }
     }
 
-    function executeL2PayloadsFromBridges() private onChain(ChainIdUtils.Ethereum()) {
-        // Base
-        chainSpellMetadata[ChainIdUtils.Base()].domain.selectFork();
-        IExecutor(Base.SPARK_EXECUTOR).execute(IExecutor(Base.SPARK_EXECUTOR).actionsSetCount() - 1);
-        // Gnosis: TODO
+    function executeForeignPayloads() private onChain(ChainIdUtils.Ethereum()) {
```

Comment:
```suggestion
    function _executeForeignPayloads() private onChain(ChainIdUtils.Ethereum()) {
```

Reply:
Maybe we could rename it to `_simulateExecuteForeignPayloads`? It's not clear by the name the intention of this function (or add a comment at the top)

Reply:
updated